src/main/java/ch/njol/skript/expressions/ExprHotbarSlot.java

```diff
@@ -33,9 +33,10 @@
 import ch.njol.skript.util.slot.Slot;
 
 @Name("Hotbar Slot")
-@Description({"The slot number of the currently selected hotbar slot."})
+@Description({"The currently selected hotbar <a href='./classes.html#slot'>slot</a>. To retrieve its number use <a href='#ExprSlotIndex'>Slot Index</a> expression."})
```

Comment:
I would like to move towards markdown in future versions for these descriptions in the future since it's easier to customise and would allow nicer descriptions.

Reply:
Strongly agree 👌